project/AppDependencies.scala

```diff
@@ -6,8 +6,8 @@ object AppDependencies {
   private val catsVersion = "2.1.1"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"       %% "auth-client"                     % "3.0.0-play-26", // To be removed when bootstrap upgrades dependencies
-    "uk.gov.hmrc"       %% "bootstrap-play-26"               % "1.14.0",
+    "uk.gov.hmrc"       %% "auth-client"                     % "3.2.0-play-26", // To be removed when bootstrap upgrades dependencies
```

Comment:
Do we need this after the bootstrap upgrade (dont think we have it in other services)?